### PR TITLE
chore: explorer feedback consistency

### PR DESCRIPTION
**Pull request by helciofranco**
- Updates icons (sync with the new website)
- Updates link structure
- Adjust typography
- Adjust spacing 
- Match new layout (almost)

| 📷 Demo |
|--------|
| Screenshot 2024-02-20 at 17 10 18  Screenshot 2024-02-20 at 17 15 23 |

### CODE REVIEW

**Review by luizstacio — APPROVED** [review]

**Review by luizstacio — CHANGES_REQUESTED** [review]
We should change the a to use Link from the UI lib?

**Comment by pedronauck** [comment]
> We should change the a to use Link from the UI lib? @pedronauck

Would be better to use it, of course

File: packages/app-explorer/src/systems/Core/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -60,92 +54,91 @@ function FooterNav({
...
+            <FooterNav title="Use" links={data.links.use} />
+            <FooterNav title="Community" links={data.links.community} />
           </Box>
-        </Box>
-      </Container>
+        </Flex>
+
+        <VStack gap="3" className={classes.social()}>
+          <HStack gap="4">
+            <a
```
Comment (luizstacio) [inline packages/app-explorer/src/systems/Core/components/Footer/Footer.tsx:76]: Can you change to use the Link from the ui library

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: packages/app-explorer/src/systems/Core/components/Footer/Footer.tsx (hunk 1 above)
Comment (helciofranco) [reply packages/app-explorer/src/systems/Core/components/Footer/Footer.tsx]: https://github.com/FuelLabs/fuel-explorer/pull/179/commits/45123d9ceeb05f7879bfca2f043497e7735d1d7a

**Review by luizstacio — APPROVED** [review]

**Pull request merged**